.github/workflows/test.yml

```diff
@@ -31,11 +28,10 @@ jobs:
           go-version: '=1.24.6'
 
       - name: Rust Toolchain
-        run: |
-          rustup toolchain install ${{ matrix.toolchain }} --profile minimal --no-self-update
-          rustup default ${{ matrix.toolchain }}
-          rustup component add rustfmt --toolchain ${{ matrix.toolchain }}
-          rustup component add clippy --toolchain ${{ matrix.toolchain }}
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          components: rustfmt,clippy
+          target: x86_64-linux-android
```

Comment:
Android needed in this workflow?

Reply:
sloppy my bad. Fixed: https://github.com/holochain/tx5/pull/195/commits/d487ef800ad24dc60d7507cc37311dbf40f94ec6